src/main/kotlin/app/cash/licensee/task.kt

```diff
@@ -263,12 +263,17 @@ internal open class LicenseeTask : DefaultTask() {
 
 internal data class PomInfo(
   val licenses: Set<PomLicense>,
+  val scm: PomScm?,
```

Comment:
Similar nesting question, and since this is an internal structure I don't think we need it even for future-proofing or anything.